tests/reference/tests/chocolateysource/add_source_all_options.rb

```diff
@@ -0,0 +1,35 @@
+require 'chocolatey_helper'
+test_name 'MODULES-3037 - Add Source With All Options'
+confine(:to, :platform => 'windows')
+
+backup_config
+
+# arrange
+chocolatey_src = <<-PP
+  chocolateysource {'test':
+    ensure   => present,
```

Comment:
Is this named 'test' intentionally or copy paste? All other manifests are named 'chocolatey'